### ISSUE TITLE
getAccount() and getPaginateMedias() doesn't work anymore.

### ISSUE THREAD

**Issue by TheSimpleCoder** [raiym/instagram-php-scraper#1067]
getAccount() and getPaginateMedias() doesn't work anymore. Need fix.

**Comment by adrianosmateus** [comment]
I changed the user-agent at ```src/InstagramScraper/Instagram.php:56``` to ```private $userAgent = 'Mozilla/5.0 (iPhone; CPU iPhone OS 10_3 like Mac OS X) AppleWebKit/603.1.23 (KHTML, like Gecko) Version/10.0 Mobile/14E5239e Safari/602.1';``` and it worked again.

**Comment by sinagolchiin** [comment]
This did not work for me.
I think Instagram used a cookie to block requests.
https://www.instagram.com/kevin/?__a=1
This request shows the following error in Chrome and Firefox browsers
for (;;); {"__ ar": 1, "error": 1357004, "errorSummary": "Sorry, something went wrong", "errorDescription": "Please try closing and re-opening your browser window.", " payload ": null," hsrp ": {" hblp ": {" consistency ": {" rev ": 1005620371}}}," lid ":" 7104639836351252026 "}
but it works for two times in edge browser 

Does anyone have a safe way to get posts on public Instagram pages?

**Comment by naearu** [comment]
I'm having the same problem.

```
    [message:protected] => Response code is 200: OK.Something went wrong. Please report issue.
    [string:Exception:private] => 
    [code:protected] => 200
    [file:protected] => /vendor/raiym/instagram-php-scraper/src/InstagramScraper/Instagram.php
    [line:protected] => 744
```

**Comment by rikless** [comment]
Looks like the _sharedData object is not available anymore :/

**Comment by tayfunyasar** [comment]
what will all crawlers do? :'(

**Comment by nathnat** [comment]
same issue here

**Comment by im-api** [comment]
> Looks like the _sharedData object is not available anymore :/

I agree too

**Comment by pierregouret** [comment]
Same Issue :cry: 
I login with basic credentials and I tested some function : 
- searchAccountsByUsername -> works
- getAccount -> doesn't work
- getAccountById -> doesn't work
- getMedias -> doesn't work
- getAccountInfo -> doesn't work

Anyone has any idea whats the bug ?

Regards

**Comment by safaksaylam** [comment]
Same issue, browser private apis not working now.

**Comment by iankaufmann** [comment]
> Same Issue 😢 I login with basic credentials and I tested some function :
> 
>     * searchAccountsByUsername -> works
> 
>     * getAccount -> doesn't work
> 
>     * getAccountById -> doesn't work
> 
>     * getMedias -> doesn't work
> 
>     * getAccountInfo -> doesn't work
> 
> 
> Anyone has any idea whats the bug ?
> 
> Regards

Instagram changed their frontend and got rid of everything that we need to make this work.

This repo (which is python) appears to have it working, with the mods listed in this issue/comment:
https://github.com/arc298/instagram-scraper/issues/805#issuecomment-1147981554

I was able to do a scrape via the command line with this updated tool.  (Although my IP got rate-limited by Instagram fairly quickly)

I assume it's just going to take some time for all the different scrapers out there to adapt to the updated Instagram UI.

**Comment by Denitz** [comment]
Yes, the `_sharedData` object has gone and `getAccount()` stopped working.

**Comment by farizaiis** [comment]
is there a solution for this without using python?

**Comment by Vivatec** [comment]
Didn't find any solution to the problem?

**Comment by rikless** [comment]
There is no solution afaik. This package has to be archived or rewrite entirely

**Comment by sagitaire123** [comment]
@rikless any other working library that you can recommend please ?

**Comment by rikless** [comment]
no :/

**Comment by SrMordred** [comment]
> I use https://github.com/pgrimaud/instagram-user-feed and it works for retrieving posts.
Well, i came here from this one because its stop working for me ;/

**Comment by rikless** [comment]
> > @rikless any other working library that you can recommend please ?
> 
> I use https://github.com/pgrimaud/instagram-user-feed and it works for retrieving posts.

Thanks, it works like a charm !

**Comment by lwiml** [comment]
https://github.com/postaddictme/instagram-php-scraper/pull/1086

**Comment by skmachine** [comment]
the APIs  https://rapidapi.com/neotank/api/instagram130/ still works fine.. weird
